quiver_engine/server.py

```diff
@@ -50,11 +59,10 @@ def get_board_files(path):
     @app.route('/inputs')
     def get_inputs():
         image_regex = re.compile(r".*\.(jpg|png|gif)$")
+        files = [filename for filename in listdir(input_folder) if image_regex.match(filename) != None ]
+        return jsonify(files)
```

Comment:
This is just reformatting?  I'd leave these couple of lines. I'm a sucker for function composition over declaring variables that only get over used once. More functional, less potentially mutable state.